Sources/SpeziFirebaseAccountStorage/FirestoreAccountStorage.swift

```diff
@@ -0,0 +1,167 @@
+//
+// This source file is part of the Stanford Spezi open-source project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University and the project authors (see CONTRIBUTORS.md)
+//
+// SPDX-License-Identifier: MIT
+//
+
+import FirebaseFirestore
+import Spezi
+import SpeziAccount
+import SpeziFirestore
+
+
+/// Store additional account details directly in firestore.
+///
+/// Certain account services, like the account services provided by Firebase, can only store certain account details.
+/// The `FirestoreAccountStorage` can be used to store additional account details, that are not supported out of the box by your account services,
+/// inside Firestore in a custom user collection.
+///
+/// - Note: The `FirestoreAccountStorage` relies on the primary [AccountId](https://swiftpackageindex.com/stanfordspezi/speziaccount/documentation/speziaccount/accountidkey)
+///     as the document identifier. Fore Firebase-based account service, this is the primary, firebase user identifier. Make sure to configure your firestore security rules respectively.
+///
+/// Once you have [AccountConfiguration](https://swiftpackageindex.com/stanfordspezi/speziaccount/documentation/speziaccount/initial-setup#Account-Configuration)
+/// and the [FirebaseAccountConfiguration](https://swiftpackageindex.com/stanfordspezi/spezifirebase/documentation/spezifirebaseaccount/firebaseaccountconfiguration)
+/// set up, you can adopt the [AccountStorageStandard](https://swiftpackageindex.com/stanfordspezi/speziaccount/documentation/speziaccount/accountstoragestandard)
+/// protocol to provide a custom storage for SpeziAccount.
+///
+/// - Important: In order to use the `FirestoreAccountStorage`, you must have [Firestore](https://swiftpackageindex.com/stanfordspezi/spezifirebase/main/documentation/spezifirestore/firestore)
+///     configured in your app. Refer to the documentation page for more information.
+///
+/// ```swift
+/// import FirebaseFirestore
+/// import Spezi
+/// import SpeziAccount
+/// import SpeziFirebaseAccountStorage
+///
+///
+/// actor ExampleStandard: Standard, AccountStorageStandard {
+///     // define the collection where you want to store your additional user data
```

Comment:
Would be great to start this comment and other comments with an upper case character and ideally use `...` to indicate that the comment continues somewhere and `.` to end a comment.
```suggestion
///     // Define the collection where you want to store your additional user data ...
```